src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -62,6 +65,8 @@ def __init__(self, cmd, name: str, location: str, exists: bool = None):
         self.cmd = cmd
         self.name = name
         self.location = _get_default_containerapps_location(cmd, location)
+        if self.location.lower() == "northcentralusstage":
+            self.location = "eastus"
```

Comment:
If we modify the location entered by users, do we need to inform the users through the log to make users aware?

Reply:
Got it, thanks!